scanners/zap/zap.py

```diff
@@ -600,7 +602,7 @@ def authentication_set_oauth2_rtoken(self):
         context_["users"] = [
             {
                 "name": Zap.USER,
-                "credentials": {"refresh_token": f"${{RTOKEN}}"},
+                "credentials": {"refresh_token": rtoken},
```

Comment:
Here we want the result to be as followed (without any replacement):

```
credential: 
  refresh_token: "${RTOKEN}"
```
To indicate that ZAP should get the token from the RTOKEN environment variable (to avoid having the token in the file itself)
However, the f-string might not be needed, actually. I think `{"refresh_token": "${RTOKEN}"}` is sufficient